android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiInitializer.java

```
@@ -35,7 +37,11 @@ public class JitsiInitializer implements Initializer<Boolean> {
     public Boolean create(@NonNull Context context) {
         Log.d(this.getClass().getCanonicalName(), "create");
 
-        SoLoader.init(context, /* native exopackage */ false);
+        try {
+            SoLoader.init(context, OpenSourceMergedSoMapping.INSTANCE);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

Why catch the exception to throw another one? Just let it bubble up.